li-hadoop-plugin/src/main/groovy/com/linkedin/gradle/liutil/LiKerberosUtil.groovy

```diff
@@ -8,12 +8,13 @@ class LiKerberosUtil {
 
   static Logger logger = LoggerFactory.getLogger(LiKerberosUtil.class);
   private static final String OOZIE_KRB5_CONF = "oozie-krb5.conf";
+  private static final String JAVA_TMP_DIR = System.getProperty("java.io.tmpdir");
 
   /**
    * We need to copy the oozie-krb5.conf file to a temporary location since we cannot point property java.security.krb5.conf to the file inside jar.
    */
   static void writeConfToTemp() {
-    logger.debug("writing oozie-krb5.conf file to ${System.getProperty("java.io.tmpdir")}");
```

Comment:
Is there something in particular that made you have to change this? I think it's good to get the temp dir at the time the method is used, rather than as a static class variable, even if this means you might call System.getProperty("java.io.tmpdir") more than once. Generally, getting things like temp directory locations at static classloading times is kind of a bad thing.